### DISCUSSION TITLE
Official CMC Minecraft Server?

### DISCUSSION THREAD

**Discussion by Bloxxify2827** [nonamecrackers2/crackers-wither-storm-mod#1566] [Q&A] [answered]
### (this is a remake of the original post, made by FireCode)

I think it would be cool to have an official Minecraft server for this mod, so that the community play together. It's kind of not hard to notice that the server advertisements channel has lots of lacking capability, due to not anyone really checking up on it. so if we just had an official main server we would actually be able to play together. Plus, there's a lot of creative and talented people in the discord, and I think it would be cool to see what everyone can build. I'm not really sure who would actually host the server, but I'm sure it wouldn't be too hard to find someone who can host.

As for the mods, i think either a Vanilla server with plugins would be okay, OR a server with a light modpack, just for performance's sake. some mods that would be good are probably some QOL ones like **Custom Player Models, Ender Trigon, Supplementaries**, Etc.

**Comment by Laughingcat27** [comment]
it would be awesome yeah, it could get a good amount of players. perhaps it could have the wither storm mod, and every now and then it gets respawned as an event?

**Comment by FireCode125** [comment]
I didn’t write that second paragraph, but yeah I think it would be nice to just have a light modpack, but obviously cwsm and ender trigon make sense to have. I might even be able to host the server, I’ll have to see what my computer can handle.

**Comment by Laughingcat27** [comment]
it should have an ingame currency and a shop system where you buy/sell items.
every player on the server will get a key daily which they can use on a crate which has a 0% chance of giving a money reward, but patrons will have a 100% chance, and the server doesn't state the chances, nor the differences between patrons and non-patrons.

**Reply by Bloxxify2827** [reply]
Okay.

**Reply by BluePineappleGuy** [reply]
the crate part just seems kinda unfair to non patrons. also, the server would be more about "oh god the phase 6 storm is here R U N" not "lemme buy some tainted blocks for a build" if its not creative. which it probably wont be since this is a survival mod, so a money/shop system isn't neccesary.

**Reply by FireCode125** [reply]
Please no

**Reply by Cod1ngGamer** [reply]
No please don't taint such an awesome server idea with the sludge that is P2W
Also this might be against the Minecraft EULA
Also if you're gonna make crates at least state the chances, otherwise you could have things like withered nether stars and command block books, and then things like steak and dirt, and people would buy the crates over and over to get the rare items, but they all have a drop chance of 0.00000000029% or something.
Some servers make people pay for these kinds of crates, and its not okay. Don't stoop down to their level.

**Comment by nonamecrackers2** [comment, marked as answer]
There'd probably be a lot of people that would want to play and it'd probably result in performance issues, I'd have to think on it

**Reply by Bloxxify2827** [reply]
there are mods/plugins to Combat performance issues, but i respect your response, hope to see what you do with this idea.

**Reply by CappyKid** [reply]
I know I'm late, but this sounds like a good idea. But it's up to Crackers.

**Reply by YassinAlaa** [reply]
> There'd probably be a lot of people that would want to play and it'd probably result in performance issues, I'd have to think on it

It's up to u! but i would prefer to play in CWSM server
